content/docs/examples/advanced-gateways/egress-gateway/index.md

```diff
@@ -29,7 +29,7 @@ on them cannot access the Internet. Defining an egress gateway, directing all th
 allocating public IPs to the egress gateway nodes allows the application nodes to access external services in a
 controlled way.
 
-{{< boilerplate before-you-begin-egress >}}
+{{< boilerplate before-you-begin-egress-gateway >}}
```

Comment:
```suggestion
{{< boilerplate before-you-begin-egress >}}

{{< warning >}}
If the egress gateway is not enabled, [enable the egress gateway](/docs/examples/advanced-gateways/egress-gateway/index.md#enable-egress-gateway).
{{< /warning >}}
```

Reply:
are you okay to have this warning for other egress tasks, which you don't necessarily need the egress gateway?

for example, I didn't want to give out the warning for this task as it is not needed:
for example:  https://preliminary.istio.io/docs/tasks/traffic-management/egress/

Reply:
Right, that's why I put the warning after the boilerplate, not inside it.

Reply:
ah good point!  thanks!!